### PR TITLE
feat(CX-1605) Admin can provide Rejection reason on a submission

### PR DESCRIPTION
**Pull request by Gon4ar5**
Resolves [CX-1605](https://artsyproduct.atlassian.net/browse/CX-1605)

Screenshot 2021-06-25 at 17 31 19
Screenshot 2021-06-25 at 17 31 34

### CODE REVIEW

**Comment by bhoggard** [comment]
Could you add a screenshot?

File: app/models/submission.rb [hunk 1]
```diff
@@ -30,6 +30,14 @@ class Submission < ApplicationRecord
     unknown_edition: 3
   }
 
+  REJECTION_REASON = [
```
Comment (olerichter00) [inline app/models/submission.rb:33]:
```suggestion
  REJECTION_REASONS = [
```

File: app/views/admin/submissions/_rejection_reason.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div class='col-md-9' id='rejected-reason'>
+  <%= @submission.rejection_reason == '' ?  'None' : @submission.rejection_reason %>
```
Comment (olerichter00) [inline app/views/admin/submissions/_rejection_reason.html.erb:2]: (question, non-blocking) Can't we just have an empty string if there's no rejection reason?

**Review by olerichter00 — APPROVED** [review]
LGTM 🎉

File: app/views/admin/submissions/_rejection_reason.html.erb (hunk 2 above)
Comment (Gon4ar5) [reply app/views/admin/submissions/_rejection_reason.html.erb:2]: Yes, I think it will be even better

Comment (olerichter00) [reply app/views/admin/submissions/_rejection_reason.html.erb]: 👍

**Comment by bhoggard** [comment]
I don't know why the tests keep failing on CircleCI with postgres unavailable. I will click the button to re-run, but go ahead and merge it once it's green. If it keeps failing, ask in `dev-help`.

**Comment by olerichter00** [comment]
It seems that the schema check failes on CircleCI:

```
convection_1           | + bundle exec rake
convection_1           | Schema IDL dumped into ./_schema.graphql
convection_1           | Files _schema.graphql and _schema.graphql.bak differ
convection_1           | schema-check failed
```

Running `rake graphql:schema:idl` should fix the issue.

**Pull request merged**